src/View/HelperTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Zend\Mvc\Plugin\FlashMessenger\View;
+
+use Zend\Mvc\Plugin\FlashMessenger\FlashMessenger as PluginFlashMessenger;
+use Zend\Mvc\Plugin\FlashMessenger\View\Helper\FlashMessenger;
+
+/**
+ * Trait HelperTrait
```

Comment:
"Trait HelperTrait" – what is this?! 😉
Can we find a short and meaningful name / description?